Shift-Demo/Shift-Demo/SplitTransitionAnimatedViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  SplitTransitionAnimatedViewController.swift
+//  Shift-Demo
+//
+//  Created by Matthew Buckley on 12/19/15.
+//  Copyright © 2015 Raizlabs. All rights reserved.
+//
+
+import UIKit
+import Shift
+
+class SplitTransitionAnimatedViewController: UITableViewController {
```

Comment:
As a rule, I declare all classes as `final` since the performance benefits of static dispatch come for free. When I need to subclass, I remove the `final` on the base class.